core/server/common/src/main/java/alluxio/ProcessUtils.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Utility methods for Alluxio processes.
+ */
+public final class ProcessUtils {
+  private static final Logger LOG = LoggerFactory.getLogger(ProcessUtils.class);
+
+  /**
+   * Runs the given process. This method should only be called from {@code main()} methods.
+   *
+   * @param process the process to run
+   * @param name the process name
+   */
+  public static void run(Process process, String name) {
+    try {
+      process.start();
+      System.exit(0);
```

Comment:
Could we get the best of both worlds by adding some of the specific info into the `toString` method of the process?